### PR TITLE
Adding parameter loglevel

### PR DESCRIPTION
**Pull request by santinoncs**

### CODE REVIEW

**Comment by rccrdpccl** [comment]
Can we parametize this from the chart?

File: chart/node-policy-webhook/values.yaml [hunk 1]
```diff
@@ -21,6 +21,11 @@ certs:
   # Key key.pem
   key: ""
 
+
+# Additional Args to command
+additionalArgs:
```
Comment (rccrdpccl) [inline chart/node-policy-webhook/values.yaml]: I would rather have `verbosity` than additional args... although this option might be useful in the future

Comment (santinoncs) [reply chart/node-policy-webhook/values.yaml]: you are right

What do you think about removing cobra?

Comment (rccrdpccl) [reply chart/node-policy-webhook/values.yaml]: If it only complicates things, we can surely remove it

**Review by rccrdpccl — APPROVED** [review]

**Comment by rccrdpccl** [comment]
Please remember to squash the PR and fix the commit message

**Pull request merged**